theme/templates/resource-landing-page/citation.html

```diff
@@ -11,551 +11,550 @@
-                                                </form>
-                                            </li>
-                                            <li data-access-type="Can edit">
-                                                <form class="share-form-edit" data-access-type="Can edit"
-                                                      method="POST">
-                                                    {% csrf_token %}
-                                                    <a>Can edit</a>
-                                                </form>
-                                            </li>
-                                            <li data-access-type="Is owner">
-                                                <form class="share-form-owner" data-access-type="Is owner"
-                                                      method="POST">
-                                                    {% csrf_token %}
-                                                    <a>Is owner</a>
-                                                </form>
-                                            </li>
-                                        </ul>
-                                    </span>
-                                </td>
-
-                                <td class="user-actions">
-                                    <form class="remove-user-form" method="POST">
-                                        {% csrf_token %}
-                                        <span class="glyphicon glyphicon-remove btn-remove-row"></span>
-                                    </form>
-                                </td>
-                            </tr>
-
-                            {# Owners with full permissions to do anything on this resource #}
-                            {% for u in owners %}
-                                <tr id="row-id-{{ u.pk }}" {% if owners|length == 1 or not is_owner_user %}class="hide-actions"{% endif %}>
-                                    <td>
-                                        <div class="user-scope">
-
-                                            {% if u.userprofile.picture %}
-                                                <div style="background-image: url('{{ u.userprofile.picture.url }}');"
-                                                class="profile-pic-thumbnail round-image"></div>
-                                            {% else %}
-                                                 <div class="profile-pic-thumbnail round-image user-icon"></div>
-                                            {% endif %}
-
-                                            <span data-col="name"
-                                                  class="user-name">{{ u|best_name }}</span>
-                                            {% if u.pk == current_user.pk %}
+        </div>
+        {% if not cm.raccess.published and is_owner_user or cm.raccess.shareable and is_view_user or cm.raccess.shareable and is_edit_user %}
```

Comment:
It may be better to put this ```if statement``` logic in ```page_processors.py``` and set a context variable ```show_manage_access``` of type bool to use here instead.